src/cli/install.sh

```diff
@@ -2,7 +2,7 @@
 set -euo pipefail
 
 if [[ ${OS:-} = Windows_NT ]]; then
-    echo 'error: Please install bun using Windows Subsystem for Linux'
+    powershell -c "irm bun.sh/install.ps1|iex"
     exit 1
```

Comment:
this exit 1 is not correct for the success case. making it 0 makes it wrong for the failure case. it should mirror what powershell exits with.

Reply:
Updated to use `$?`.